docs/manual/scala/guide/cluster/index.toc

```diff
@@ -4,6 +4,8 @@ PersistentEntityRDBMS:Relational Database Persistent Entities
 ReadSide:Persistent Read-Side
 ReadSideCassandra:Cassandra Read-Side Support
 ReadSideRDBMS:Relational Database Read-Side Support
+ReadSideJDBC:JDBC Read-Side Support
+ReadSideSlick:Slick Read-Side Support
```

Comment:
There's an under-documented feature of typesafe-markdown allowing a list of `next-links` which we could use here:

```
ReadSideRDBMS:Relational Database Read-Side Support;next=ReadSideJDBC,ReadSideSlick
ReadSideJDBC:JDBC Read-Side Support
ReadSideSlick:Slick Read-Side Support
```
with this change the list of links at the bottom of `ReadSideRDBMS` can be removed.